src/App.js

```diff
@@ -1,13 +1,19 @@
 import React from "react";
+import Heading from "./Heading"
+import TouristInfoCards from "./TouristInfoCards"
+import Footer from "./Footer"
 
 import Bookings from "./Bookings";
 import "./App.css";
 
 const App = () => {
+  const footerContentArray = ["123 Fake Street, London, E1 4UD", "hello@fakehotel.com", "0123 456789"]
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Footer footercontent={footerContentArray} />
```

Comment:
Because this `prop` is specific to the `Footer`, it's kind of unnecessary to call it `footercontent` - we could just call it `content` (we're giving it to the `Footer`, so we don't need to say `Footer` again).

But in _this_ component, `footerContentArray` is a good variable name, because _this_ component deals with many child components.